examples/config/base/go-kprobe-counter/bytecode.yaml

```diff
@@ -15,4 +15,4 @@ spec:
   bytecode:
     image:
       url: quay.io/bpfman-bytecode/go-kprobe-counter:latest
-      imagepullpolicy: Always
+      imagepullpolicy: IfNotPresent
```

Comment:
Why the change here? For CI at least Always doesn't seem like a bad thing

Reply:
Good question.  I changed it because it's not specified on the other existing programs, and I thought that the default was IfNotPresent.  IfNotPresent is also faster.  However, I just checked and the default is actually [it depends on several factors](https://kubernetes.io/docs/concepts/containers/images/).  So, I think we should explicitly specify it the same way for all the program types, and I agree with IfNotPresent so you always get the latest image with the given tag.